scripts/xacro

```diff
@@ -1,33 +0,0 @@
-#! /usr/bin/env python3
```

Comment:
no longer needed. a similar wrapper script (and executable that calls it) is automatically generated by the `setuptools`